tests/sles4sap/ipaddr2/deploy.pm

```diff
@@ -52,6 +44,24 @@ sub run {
     # break test steps that relay to remote ssh comman output
     assert_script_run('rm ~/.ssh/config');
 
+    my $os;
+    if (get_var('PUBLIC_CLOUD_IMAGE_LOCATION')) {
+        # This section is only needed by Azure tests using images uploaded
+        # with publiccloud_upload_img. This is because qe-sap-deployment
+        # is still not able to use images from Azure Gallery
```

Comment:
The overall sentence `This is because qe-sap-deployment is still not able to use images from Azure Gallery`  can be removed